### ISSUE TITLE
Server crashes right after startup // TileMultipart_cmp$$24 ...  ILogisticalTransporter

### ISSUE THREAD

**Issue by chaorx** [aidancbrady/Mekanism#2616]
# Mekanism-1.7.10-8.1.2.229 - Server crashes right after startup.

## Beginning of stacktrace:

---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 5/2/15 5:06 PM
Description: Ticking block entity

java.lang.ClassCastException: TileMultipart_cmp$$24 cannot be cast to mekanism.common.base.ILogisticalTransporter
	at mekanism.common.content.transporter.TransporterPathfinder$IdlePath.find(TransporterPathfinder.java:74)
	at mekanism.common.content.transporter.TransporterPathfinder.getIdlePath(TransporterPathfinder.java:550)

## Dump of "crash-2015-05-02_17.06.01-server.txt"
http://pastebin.com/TdyH99Fc

**Comment by beringtom** [comment]
Try updating CodeChicken core and his other mods you might use. You are running an old version.

**Comment by chaorx** [comment]
Thank you for the quick reply, its a great mod you make!

Updated Chicken Bones Mods latest versions. I run NEI, Core and Enderstorage, along with the auto-downloading CodeChickenLib-1.7.10-1.1.3.136-universal.jar.

The issue persists.  

http://pastebin.com/18ZuPXXw

First try with the new files I got the server up, but once I tried to connect a client it went down.
After that first start it crashes right after startup seems to have completed before connecting a client.

/Chaorx
